index.html

```diff
@@ -42,6 +42,148 @@ <h2 class="headline-fmt supporting-text">I am a Software Developer</h2>
+          </div>
+          <div class="content-defaults content3">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content4">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content5">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+          <div class="content-defaults content6">
+            <h2 class="other-content-title">Profesional Art Printing Data</h2>
+            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard</p>
+            <ul class="other-content-tags">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <a class="other-content-link" href="#" target="_blank">See Project</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="about">
+      <div class="about-top">
+        <h1 class="about-primary-text">About me</h1>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+        <a class="about-link" href="#" target="_blank">Get My Resume</a>
+      </div>
+
+      <hr class="about-hr">
+      <div class="about-bottom">
+        <ul>
+          <li class="about-bottom-subtopic subtopic-language">Language</li>
```

Comment:
I think you should reduce the font size of the sub-topics to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.
 *Figma Design*👇
![font](https://user-images.githubusercontent.com/73854999/198829042-854979bc-0ecf-4df1-b93a-fb1b67d7faed.PNG)

*Your design* 👇
![fonts](https://user-images.githubusercontent.com/73854999/198829054-5b6104fe-112b-476a-b9d9-b25b08379cfa.PNG)